charts/core/src/ChartCard/ChartCard.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import {
+  borderRadius,
+  color,
+  InteractionState,
+  spacing,
+  transitionDuration,
+  Variant,
+} from '@leafygreen-ui/tokens';
+
+export const getContainerStyles = (theme: Theme) => css`
+  border: 1px solid
+    ${color[theme].border[Variant.Disabled][InteractionState.Default]};
+  border-radius: ${borderRadius[200]}px;
+  overflow: hidden;
+  width: 100%;
+  display: grid;
+  grid-template-rows: 40px 0fr;
+  transition: grid-template-rows ${transitionDuration.slower}ms ease-in-out;
+
+  &.open {
```

Comment:
Curious: why do you prefer using `.open` instead of conditionally rendering the styles with `cx`?

Reply:
Actually not a preference, just not used to having `cx` as an option! Will update this.